build/xtask/src/dist.rs

```diff
@@ -1311,8 +1338,8 @@ fn generate_task_linker_script(
 
     writeln!(linkscr, "MEMORY\n{{")?;
     for (name, range) in map {
```

Comment:
I think this'd be clearer if the name became `ranges`, since it's now more than one range.

Reply:
Yup, done in 05e21fbee78db4408a70b4ddc01935e2061316c9